src/components/Link/ButtonLink/__tests__/ButtonLink.spec.jsx

```diff
@@ -4,6 +4,7 @@ import { mount, render } from 'enzyme'
 import { warn } from '../../../../utils/warn'
 
 import ButtonLink from '../ButtonLink'
+import Responsive from '../../../Responsive/Responsive'
```

Comment:
I don't see a "ButtonLink.ssr.spec.jsx" file..... I think this test should be rewritten to follow the SSR pattern established in the other components.